core/config/config.go

```diff
@@ -0,0 +1,91 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+// File defines the fields required to point to a local config file
+type File struct {
+	// Name of the config file (without extension)
+	Name string
+
+	// Extension of the config file.
+	//
+	// REQUIRED if the config file does not have the extension in the name
+	Type string
+
+	// Path to look for the config file in
+	Path string
+}
+
+// Handler represents a wrapper around Viper
+type Handler struct {
+	vp *viper.Viper
```

Comment:
I'm not sure what viper brings for us?
What is the value?

Reply:
I had used it in the POC initially, just for simplicity.

Its no longer a part of my current updated implementation.